client/src/components/trackInteractions/ClickTracker.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import axios from 'axios';
+
+//https://reactjs.org/docs/higher-order-components.html
+// HOC for click tracker
+
+function ClickTracker (WrappedComponent) {
+
+  return class extends React.Component {
+    constructor(props) {
+      super(props);
+
+      this.postTrackInteractions = this.postTrackInteractions.bind(this);
+    }
+      postTrackInteractions(elementName, widgetName) {
+
+        axios.post('/api/interactions', {
```

Comment:
I was wondering if this has been tested, it looks like the API token is not included yet

Reply:
Nevermind, this is the client, not the server